content/docs/1.13/client-libraries.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Client libraries
+title: Client Libraries
 weight: 5
 children:
-- title: Client features
+- title: Client Features
```

Comment:
How about just "Features"? It's under "Client Libraries" already.

Reply:
I tried that previously, but we have another Features page and it looked slightly confusing.